src/components/pages/DashboardPage/DashboardPage.css

```diff
@@ -0,0 +1,6 @@
+.dashboard-page {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
```

Comment:
Maybe subjected to change if the left panel is separate from the dashboard for graph info

Reply:
We'll come back and change it once we have all the components on the page.